games-arcade/rocksndiamonds/rocksndiamonds-4.0.0.2.recipe

```diff
@@ -48,14 +48,15 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make sdl RW_GAME_DIR=$settingsDir/rocksndiamonds
+	make #RW_GAME_DIR=$settingsDir/rocksndiamonds
```

Comment:
@pulkomandy is it that bad of an idea to leave them in ~/.rocksndiamonds, I see more stuff still leaving their settings files in there?